applications/FluidDynamicsApplication/custom_processes/weighted_divergence_calculation_process.h

```diff
@@ -0,0 +1,338 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Riccardo Tosi
+
+#ifndef KRATOS_WEIGHTED_DIVERGENCE_CALCULATION_PROCESS_H
+#define KRATOS_WEIGHTED_DIVERGENCE_CALCULATION_PROCESS_H
+
+
+#include "includes/define.h"
+#include "includes/model_part.h"
+#include "includes/element.h"
+#include "includes/kratos_flags.h"
+#include "processes/process.h"
+#include "geometries/geometry.h"
+#include "utilities/geometry_utilities.h"
+#include "utilities/math_utils.h"
+#include "includes/kratos_parameters.h"
+#include "utilities/variable_utils.h"
+
+#include <string>
+#include <iostream>
+#include <sstream>
+
+#include <boost/functional/hash.hpp> //TODO: remove this dependence when Kratos has en internal one
```

Comment:
Why do you need all these?. As a comment aside, we are trying to reduce the Boost dependencies as much as possible.

Reply:
Removed